packages/ui-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -33,13 +34,23 @@ const Checkbox: React.FC<Props> = ({ label, name, onChange, header, checked, val
         </div>
       ) : null}
       <div className={styles.row}>
-        <input name={name} type="checkbox" id={id} value={value} onChange={onChange} checked={checked} aria-required={required} disabled={disabled} />
+        <input
+          name={name}
+          type="checkbox"
+          id={id}
+          value={value}
+          onChange={onChange}
+          checked={checked}
+          disabled={disabled}
+          aria-required={required}
```

Comment:
`required` is preferred above `aria-required`, because this element supports the native `required` attribute

(although this was already there in the old situation, lets make it a little bit better)

Reply:
Ignore my above comment, i think it's better to keep it at aria-required because we can have a custom validation message instead of the browser doing the "required" validation